tests/ZttpTest.php

```diff
@@ -487,6 +487,14 @@ function can_use_digest_auth()
         $this->assertTrue($response->isOk());
     }
 
+    /** @test */
+    function can_retrieve_effective_uri()
```

Comment:
It misses `public` hint for this method.

Reply:
I'm all for adding the `public` hint, but wouldn't that make it inconsistent with the other tests, considering that they don't have any visibility hints declared either?